authz/providers/azure/rbac/checkaccess_v2.go

```diff
@@ -302,7 +302,7 @@ func (a *AccessInfo) checkAccessV2(ctx context.Context, request *authzv1.Subject
 	if a.fleetManagerResourceId != "" {
 		log.V(7).Info("Falling back to fleet manager scope check (v2)", "fleetResourceId", a.fleetManagerResourceId)
 
-		fleetResourceId, err := buildResourceIDForV2(a.fleetManagerResourceId, managedNamespaceExists, managedNamespacePath)
+		fleetResourceId, err := buildResourceIDForV2(a.fleetManagerResourceId, namespaceExist, namespaceString)
```

Comment:
The PR description says

"When fleetResourceID exists, always check against the fleet scope first. Then if the request is namespace scoped, check against fleet managedNamespace secondly."

by "fleet scope," do you mean "/fleets/test-fleet/namespaces/test-ns" or "/fleets/test-fleet"

The description seems to imply the latter but code seems to imply the former

Reply:
It depends on the request, if the request is namespace scoped, it checks `/fleet/test-fleet/namespace/test-ns`; otherwise, it checks `/fleet/test-fleet`.